src/reanimated2/Colors.ts

```diff
@@ -7,18 +7,29 @@
  */
 
 /* eslint no-bitwise: 0 */
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-nocheck
 import { Platform } from 'react-native';
 import { makeRemote, makeShareable, isConfigured } from './core';
 import { interpolate } from './interpolation';
+// @ts-ignore JS file
 import { Extrapolate } from '../reanimated1/derived';
 
+type RBGObj = {
+  r: number;
+  g: number;
+  b: number;
+};
+
+type HSVObj = {
+  h: number;
+  s: number;
+  v: number;
+};
```

Comment:
Is there a specific reason for making those things `type`? `interface`s have a few advantages over types, IIRC that's for performance and decl merging.

so,

```ts
interface RGB {
  r: number;
  g: number;
  b: number;
}
```


is always better unless you have a specific reason

Reply:
Mosty muscle memory after I stumbled upon this issue: https://github.com/microsoft/TypeScript/issues/15300

This code can easily be swapped for interfaces, thanks for noticing.

Reply:
Fixed in c2dfc60.